dependencies.props

```diff
@@ -9,19 +9,19 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
-    <CoreFxCurrentRef>29a6592b687e8284ebba360327f48e9e6af5d0cd</CoreFxCurrentRef>
-    <CoreClrCurrentRef>29a6592b687e8284ebba360327f48e9e6af5d0cd</CoreClrCurrentRef>
-    <StandardCurrentRef>99e7157bcbef063e89b0fe6a0f40ad509d8d8722</StandardCurrentRef>
+    <CoreFxCurrentRef>b250d530b92cb23ff7412c1560285f52b3a1979b</CoreFxCurrentRef>
+    <CoreClrCurrentRef>b250d530b92cb23ff7412c1560285f52b3a1979b</CoreClrCurrentRef>
+    <StandardCurrentRef>b250d530b92cb23ff7412c1560285f52b3a1979b</StandardCurrentRef>
     <WCFCurrentRef>c1ea043aefee0ebcc56c298358953eaf26ff985a</WCFCurrentRef>
   </PropertyGroup>
 
   <PropertyGroup>
-    <PlatformPackageVersion>2.0.1</PlatformPackageVersion>
-    <MicrosoftPrivateCoreFxNETCoreAppPackageVersion>4.4.2-servicing-25814-01</MicrosoftPrivateCoreFxNETCoreAppPackageVersion>
+    <PlatformPackageVersion>2.0.0</PlatformPackageVersion>
```

Comment:
@safern - why isn't the Platform package version bumped?  See https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest_Packages.txt#L2.  It says we are still producing `2.0.0`.

Reply:
It is because we are doing two versions at the same time and we haven't fully published the other one yet. This shouldn't be any issue right now and should work itself out after we release the other.